data-access/basic_data_access.ipynb

```diff
@@ -167,538 +248,1117 @@
-    "# Next, export the data to a netcdf file \n",
-    "ck.export(data_to_download, filename=\"my_filename1\", format=\"NetCDF\") "
+    "# Let's plot what the 1-in-X return values look like over Alameda County\n",
+    "for x in return_periods:\n",
+    "    fig,ax = plt.subplots()\n",
+    "    one_in_x_data.mean(dim='sim').sel(one_in_x=x).return_values.plot(\n",
+    "        cmap='plasma', \n",
+    "        x='lon', \n",
+    "        y='lat',\n",
+    "        cbar_kwargs={'label': f'1-in-{x} year Daily Max Temperature (°F)'}\n",
+    "    )\n",
+    "    ax.set_title(f'1-in-{x} Year Daily Max Temp averaged \\nover all WRF sims for Alameda County')\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5df5827-549f-4c90-977c-79e63557c836",
+   "metadata": {},
+   "source": [
+    "### Processor Example 9: Bias Adjust Model to Station"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "32306bec",
+   "id": "b1d818ce",
    "metadata": {},
    "source": [
-    "Some additional file export formats are demonstrated below. The code has been commented out; simply remove the code comment hash (#) at beginning of the line to make the code executable. "
+    "In case you want to station bias correct gridded climate model data to historical observations from weather stations, you can pass in your desired weather station to the `bias_adjust_model_to_station` processor that we provide:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "47856284",
+   "id": "536c54b2",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ck.export(data_to_use, filename=\"my_filename2\", format=\"Zarr\") # Zarr export locally\n",
+    "# Let's manually bias adjust our climate data to KSAC (the SAC airport weather station).\n",
+    "manual_bias_adjustment = (cd\n",
+    "    .catalog(\"cadcat\")\n",
+    "    .activity_id(\"WRF\")\n",
+    "    .institution_id(\"UCLA\")\n",
+    "    .table_id(\"1hr\")\n",
+    "    .grid_label(\"d02\")\n",
+    "    .variable(\"t2\")\n",
+    "    .processes({\n",
+    "        \"bias_adjust_model_to_station\": {\n",
+    "            \"stations\": ['KSAC'],\n",
+    "            \"historical_slice\": (1980, 2014), # You can specify what historical period to bias correct for.\n",
+    "        }\n",
+    "    })\n",
+    "    .get()\n",
+    ")\n",
     "\n",
-    "# ck.export(data_to_use, filename=\"my_filename3\", format=\"Zarr\", mode=\"s3\") # Zarr export to S3\n",
+    "manual_bias_adjustment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "581676cf",
+   "metadata": {},
+   "source": [
+    "### Processor Example 10: Export"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b8f682b-3972-4235-b09c-1eab5017fdbf",
+   "metadata": {},
+   "source": [
+    "Now that we’ve walked through the processors above, you may want to take the data you’ve been working with and move it elsewhere. The `export` processor makes this straightforward.\n",
```

Comment:
'take the data you’ve been working with and move it elsewhere' feels vague, could be more specific that export means we're showing you how to write your data to file